### PR TITLE
Update installation and documentation

### PR DESCRIPTION
**Pull request by wensi-ai**
1. Clean up and move all scripts to scripts folder
2. Automated documentation update (deployment here https://stanfordvl.github.io/OmniGibson/)
3. Fixed gh checking bugs

### CODE REVIEW

File: omnigibson/macros.py [hunk 1]
```diff
@@ -13,11 +13,13 @@
 macros = Dict()
 gm = macros.globals
 
-# Paths (either relative to this directory or global absolute paths) for assets
+# Path (either relative to OmniGibson root directory or global absolute path) for data
```
Comment (wensi-ai) [inline omnigibson/macros.py:16]: this is relative to OmniGibson/omnigibson. I'll change it later.

**Review by ChengshuLi — APPROVED** [review]
1. setup_conda_env.sh should change isaac_sim.bat to isaac_sim.sh
2. setup_conda_env.sh should respect custom conda env name
I roughly went over this PR. @cgokmen @cremebrule If one of you could also take a look, that would be great!

File: omnigibson/macros.py (hunk 1 above)
Comment (wensi-ai) [reply omnigibson/macros.py]: done

File: docs/getting_started/building_blocks.md [hunk 2]
```diff
@@ -277,10 +277,7 @@ This demo loads an apple and a knife, and showcases how apple can be diced into
 python -m omnigibson.examples.object_states.folded_unfolded_state_demo
 ```
 
-This demo loads in three different cloth objects, and allows you to manipulate them (1) while printing out their `Folded` state status in real-time.
-{ .annotate }
-
-1. Manipulate the object by holding down **`Shift`** and then **`Left-click + Drag`**!
+This demo loads in three different cloth objects, and allows you to manipulate them while printing out their `Folded` state status in real-time. Try manipulate the object by holding down **`Shift`** and then **`Left-click + Drag`**!
```
Comment (cremebrule) [inline docs/getting_started/building_blocks.md:280]: typo: "try manipulating"

File: docs/getting_started/building_blocks.md [hunk 3]
```diff
@@ -371,10 +368,7 @@ This demo loads in a stove (toggled on) and two apples. The first apple will be
 python -m omnigibson.examples.object_states.overlaid_demo
 ```
 
-This demo loads in a carpet on top of a table. The demo allows you to manipulate the carpet (1) while printing out their `Overlaid` state status in real-time.
-{ .annotate }
-
-1. Manipulate the object by holding down **`Shift`** and then **`Left-click + Drag`**!
+This demo loads in a carpet on top of a table. The demo allows you to manipulate the carpet while printing out their `Overlaid` state status in real-time. Try manipulate the object by holding down **`Shift`** and then **`Left-click + Drag`**!
```
Comment (cremebrule) [inline docs/getting_started/building_blocks.md:371]: same

File: docs/getting_started/installation.md [hunk 4]
```diff
@@ -217,3 +195,14 @@ python -m omnigibson.examples.robots.robot_control_example # (1)!
...
 **Next:** Get quickly familiarized with **`OmniGibson`** from our [Quickstart Guide](./quickstart.md)!
+
+
+## 🧯 **Troubleshooting**
+
+??? question "I cannot open Omniverse Launcher AppImage on Linux"
+
+    You probably need to [install FUSE](https://github.com/AppImage/AppImageKit/wiki/FUSE) to run the Omniverse Launcher AppImage.
+
+??? question "OmniGibson stucks at shader compilation upon startup"
```
Comment (cremebrule) [inline docs/getting_started/installation.md:206]: typo: "is stuck"

File: omnigibson/macros.py [hunk 5]
```diff
@@ -13,11 +13,13 @@
 macros = Dict()
 gm = macros.globals
 
-# Paths (either relative to this directory or global absolute paths) for assets
+# Path (either relative to OmniGibson/omnigibson directory or global absolute path) for data
+gm.DATA_PATH = "data"
```
Comment (cremebrule) [inline omnigibson/macros.py:17]: there's a possibility that assets / dataset may be downloaded into different places, but it probably won't matter for now

File: scripts/download_datasets.py [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+os.environ["OMNIGIBSON_NO_OMNIVERSE"] = "1"
+
+from omnigibson.macros import gm
+from omnigibson.utils.asset_utils import download_og_dataset, download_assets
+import click
+
+
+def main():
+    # Ask user which dataset to install
+    data_path = gm.DATA_PATH if os.path.isabs(gm.DATA_PATH) else os.path.abspath(f"omnigibson/{gm.DATA_PATH}")
```
Comment (cremebrule) [inline scripts/download_datasets.py:14]: IIRC these data paths should already be modified at runtime so that they're always an absolute path

**Review by cremebrule — COMMENTED** [review]
some small comments, overall lgtm!!

File: docs/getting_started/installation.md (hunk 4 above)
Comment (wensi-ai) [reply docs/getting_started/installation.md:206]: done

File: docs/getting_started/building_blocks.md (hunk 3 above)
Comment (wensi-ai) [reply docs/getting_started/building_blocks.md:371]: done

File: docs/getting_started/building_blocks.md (hunk 2 above)
Comment (wensi-ai) [reply docs/getting_started/building_blocks.md:280]: done

File: scripts/download_datasets.py (hunk 6 above)
Comment (wensi-ai) [reply scripts/download_datasets.py]: reverted change

File: omnigibson/macros.py (hunk 5 above)
Comment (wensi-ai) [reply omnigibson/macros.py]: reverted change

**Pull request merged**